### PR TITLE
Add button component for login, register & back

### LINKED ISSUE #12: Components
#### Description

List of components that need to be built and what is needed for them.

#### Tasks

- [x] Component Link for LandingPage

- [x] Component Logo with Dog Picture and Title "DogDay" with background and shadow

- [x] Component LogoImg with Dog Picture and Title

- [x] Component Button with label and shadow

- [x] Component BackButton blue with arrow

- [x] Component NavBar
1. Logo
2. Three yellow dots with shadow with active effect blue that link to the pages

- [x] Component Dog Card with shadow
1. Profile picture 
2. dog name
3. dog age

- [x] Component Decision board
1. Yellow "Yes" and "No" button with shadow to decide whether I like the dog/sitter or not. 
2. blue background with shadow

- [x] Component LabledInput for Register and LogIn Form

- [x] Component ProfileCard
1. Info Component with Email, Handynummer, Stadt, Hundealter
2. Background circle in blue (sitter) or yellow (dog)
3. Picture
4. Name in the corresponding circle color

- [x] Component MatchCardDog
1. Card with dog image and info (name, age, contact details)

- [x] Component MatchCardSitter 
1. Card with dog image and info (name, experience, contact details)

### PR DESCRIPTION
**Pull request by lajanzen**
Created:
- register buttons
- log-in button
- back button

Added color and font variables to global.css and imported global.css to storybook.

### CODE REVIEW

**Review by Dev-dfm — APPROVED** [review]
lgtm 👍

**Review by MaxMuecket — APPROVED** [review]
lgtm ✌️

Maybe you can add a description or issue.

**Review by daebert — APPROVED** [review]
lgtm!

**Review by Niklas-Schaefers — APPROVED** [review]
lgtm 🚀

File: src/app/components/BackButton/BackButton.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React, { ReactNode } from "react";
+import styles from "./BackButton.module.css";
+
+type BackButtonProps = {
+  children: ReactNode;
+};
+
+function BackButton({ children }: BackButtonProps): JSX.Element {
```
Comment (lmachens) [inline src/app/components/BackButton/BackButton.tsx]: Actually, this is not a `BackButton`, it is a `Button`.
You can use this component to create a `BackButton` (which might have an `onClick` and specific children`).

File: src/app/components/LogInButton/LogInButton.module.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.logInButton {
```
Comment (lmachens) [inline src/app/components/LogInButton/LogInButton.module.css]: This is very similar to the `registerButton`. Try to find more generic names for the components like `PrimaryButton` and `SecondaryButton` to allow usage in different scenarios. You might want to reuse this component somewhere else.

I would merge all buttons to a single component and change the CSS BEM like:

```css
.button {
  // Base CSS
}

.button--primary {
  background-color: var(--primary-color);
}

.button--secondary {
  background-color: var(--secondary-color);
}

.button--round {
  border-radius: 50%; 
  // more styles for the round button you like to use as back button
}
```

**Review by lmachens — COMMENTED** [review]
good start.

**Review by infoucarsinan — APPROVED** [review]

**Pull request merged**